src/main/java/ua/ukr/net/model/Employee.java

```diff
@@ -89,7 +81,6 @@ public String toString() {
                 ", name='" + name + '\'' +
```

Comment:
it is better to use com.google.gson library and override toString() like:
return new Gson().toJson(this);